src/components/OsaamisSuosittelija/OsaamisSuosittelija.tsx

```diff
@@ -72,20 +70,10 @@ export const OsaamisSuosittelija = ({ description, value = [], onChange }: Osaam
 
   return (
     <div className={`mb-6 ${sm ? 'grid grid-cols-3 gap-x-5' : 'flex flex-col'}`}>
-      <div className={`${sm ? 'col-span-1' : 'order-1'} flex items-end`}>
-        <label htmlFor={ehdotetutOsaamisetId} className="mb-4 inline-block align-top text-form-label text-black">
-          {t('work-history.proposed-competences')}
-        </label>
-      </div>
-      <div className={`${sm ? 'col-span-2' : 'order-3'} flex items-end`}>
-        <label htmlFor={valitsemasiOsaamisetId} className="mb-4 inline-block align-top text-form-label text-black">
-          {t('work-history.competences-of-your-choice')}
-        </label>
-      </div>
       <div
-        className={`${sm ? 'col-span-1' : 'order-2 mb-6'} h-[400px] overflow-y-auto rounded-[10px] border-[5px] border-solid border-border-gray p-5`}
+        className={`${sm ? 'col-span-1' : 'order-2 mb-6'} h-[400px] overflow-y-auto rounded-[8px] border border-border-gray p-5 bg-white`}
```

Comment:
Use `rounded` or `rounded-md` class

Reply:
This is now changed to use just `rounded`.